.gitlab-ci/packet.yml

```diff
@@ -52,7 +52,7 @@ packet_ubuntu20-aio-docker:
   when: on_success
 
 packet_ubuntu20-calico-aio-hardening:
-  stage: deploy-part2
+  stage: unit-tests
```

Comment:
After verifying the change works for cert-manager at CI, this part will be reverted.

Reply:
By specifying securityContext for cert-manager-cainjector deployment, there is the corresponding pod.
But there are not any pods for the other cert-manager deployments which are not specified with the securityContext.
```
ok: [instance-1] => {
    "msg": [
        "NAMESPACE      NAME                                       READY   STATUS    RESTARTS      AGE     IP               NODE         NOMINATED NODE   READINESS GATES",
        "cert-manager   cert-manager-cainjector-79bfdbf497-6x6sj   1/1     Running   0             91s     10.233.87.129    instance-1   <none>           <none>",
        "default        netchecker-server-5857c6bc77-mxck7         2/2     Running   1 (59s ago)   63s     10.233.87.132    instance-1   <none>           <none>",
        "kube-system    calico-kube-controllers-56fd7b8dc-s9wmp    1/1     Running   0             97s     172.30.113.207   instance-1   <none>           <none>",
        "kube-system    calico-node-zlswn                          1/1     Running   0             116s    172.30.113.207   instance-1   <none>           <none>",
        "kube-system    coredns-74d6c5659f-6nzzb                   1/1     Running   0             76s     10.233.87.130    instance-1   <none>           <none>",
        "kube-system    dns-autoscaler-6656dfd4c6-fvt9m            1/1     Running   0             72s     10.233.87.131    instance-1   <none>           <none>",
        "kube-system    etcd-instance-1                            1/1     Running   0             2m47s   172.30.113.207   instance-1   <none>           <none>",
        "kube-system    kube-apiserver-instance-1                  1/1     Running   1             2m47s   172.30.113.207   instance-1   <none>           <none>",
        "kube-system    kube-controller-manager-instance-1         1/1     Running   1             2m47s   172.30.113.207   instance-1   <none>           <none>",
        "kube-system    kube-proxy-zsvw7                           1/1     Running   0             117s    172.30.113.207   instance-1   <none>           <none>",
        "kube-system    kube-scheduler-instance-1                  1/1     Running   1             2m47s   172.30.113.207   instance-1   <none>           <none>"
    ]
}
```